motion_tracker/model/model.py

```diff
@@ -57,6 +64,8 @@ def make_model(img_edge_size, backend_id):
     start_box_mask_layer = start_box_mask
     for layer in pooling_layers:
         start_box_mask_layer = layer(start_box_mask_layer)
+    start_box_mask_layer = MaxPooling2D(pool_size=(3, 3), 
+                         dim_ordering=backend_id)(start_box_mask_layer)
 
```

Comment:
This is the only surprising thing that I had in to get this to run. I haven't sat down to fully understand it, but currently it works.